playbooks/rhv/inventory.example

```diff
@@ -0,0 +1,61 @@
+[all:vars]
```

Comment:
Why did you remove openshift_release?

Reply:
I believe it defaults to current (pertinent for master branch), and it is not specific to RHV